packages/homepage/src/pages/explore.js

```diff
@@ -74,8 +74,7 @@ export default class Explore extends React.PureComponent {
   openSandbox = index => {
     const sandbox = this.state.sandboxes[index];
     const { id, picks } = sandbox;
-    const title = picks[0].title;
-    const description = picks[0].description;
+    const { title, description } = picks[0];
```

Comment:
or... 😉

```js
 const [{ title, description }] = picks;
```

Reply:
Even cleaner :record_button: :+1:

Reply:
i just hate seeing array access with numeric literals when you can avoid it. it's one of my red flags when i'm reviewing at code.